### PR TITLE
added API requests

### PR DESCRIPTION
**Pull request by alexgul1**

### CODE REVIEW

**Comment by matroskin062** [comment]
Maybe I got it wrong, but what’s the reason to use async without await?
Perhaps, it’s better to rewrite this functions using async/await syntax?

**Comment by alexgul1** [comment]
> Maybe I got it wrong, but what’s the reason to use async without await?
> Perhaps, it’s better to rewrite this functions using async/await syntax?

is axios an asynchronous function? if so, then maybe i remove the construction of the asynchronous function?

**Comment by matroskin062** [comment]
> > Maybe I got it wrong, but what’s the reason to use async without await?
> > Perhaps, it’s better to rewrite this functions using async/await syntax?
> 
> is axios an asynchronous function? if so, then maybe i remove the construction of the asynchronous function?

You can just remove `async` keyword or do somethnig like this:

```javascript
const getData2 = async () => {
  try {
    const res = await fetch('url');
    const json = await res.json();
    return json;
  } catch (e) {
    return e;
  }
};
```

Both options are valid

**Comment by alexgul1** [comment]
> > > Maybe I got it wrong, but what’s the reason to use async without await?
> > > Perhaps, it’s better to rewrite this functions using async/await syntax?
> > 
> > 
> > is axios an asynchronous function? if so, then maybe i remove the construction of the asynchronous function?
> 
> You can just remove `async` keyword or do somethnig like this:
> 
> ```js
> const getData2 = async () => {
>   try {
>     const res = await fetch('url');
>     const json = await res.json();
>     return json;
>   } catch (e) {
>     return e;
>   }
> };
> ```
> 
> Both options are valid

OK, I'll do it now without async keyword, then if we need another variation i'll change it

**Comment by alexgul1** [comment]
UPDATED

File: src/js/AuthAPI.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import User from './User';
+import axiosInstance from './config';
+
+export default class AuthAPI {
+  static login({ identifier, password }) {
+    console.log(axiosInstance.headers);
```
Comment (matroskin062) [inline src/js/AuthAPI.js]: Please remove this unnecessary console log

**Review by matroskin062 — APPROVED** [review]

File: src/js/AuthAPI.js (hunk 1 above)
Comment (alexgul1) [reply src/js/AuthAPI.js]: done

**Review by matroskin062 — APPROVED** [review]

**Comment by matroskin062** [comment]
Looks good 👍

**Review by muartem — APPROVED** [review]

**Pull request merged**